microsoft-edge/webview2/concepts/browser-features.md

```diff
@@ -200,7 +200,7 @@ If you set `AreBrowserAcceleratorKeysEnabled` to `FALSE`, the following addition
 | Find Previous | **Shift+F3** |
 | Back | **Alt+Left Arrow**, **Browser Back Button Key** |
 | Forward | **Alt+Right Arrow**, **Browser Forward Key** |
-| Print | **Ctrl+P** |
+| Caret Browsing | **F7** |
```

Comment:
i'll check the wording "Caret Browsing" to see if standard.
GH rendered section:
https://github.com/MicrosoftDocs/edge-developer/blob/user/niburman/update-browser-features/microsoft-edge/webview2/concepts/browser-features.md#shortcuts-turned-off-if-acceleratorenabled-is-false

Reply:
this thread can be resolved